File: C/unit_2/Fa.c
#include <stdio.h>
int main()
{
    int num, temp, sum = 0, rem;
    printf("Enter the number : \n");
    scanf("%d", &num);
    temp = num;
    while (temp != 0)
    {
        rem = temp % 10;
        sum = sum + rem;
        temp /= 10;
    }
    printf("The sum of digit is : %d", sum);
    return 0;
}

File: Java/unit_2/Fa.java
package Java.unit_2;

import java.util.Scanner;

public class Fa {
    public static void main(String[] args) {
        int sum = 0, m, n;
        System.out.println("Enter the Number : \n");
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        while (n > 0) {
            m = n % 10;
            sum += m;
            n /= 10;
        }
        System.out.println("The Sum of digits of the number : " + sum);
        sc.close();
    }

}
